components/internal-services/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+resources:
```

Comment:
please also include README.md (at leaste minimal) in components/internal-services.

Reply:
added